config/routes.rb

```diff
@@ -14,7 +15,9 @@
       get 'description'
       get 'amenities'
       get 'location'
+      get 'photos'
     end
+    resources :reservations, only: [:create]
```

Comment:
put this outside of resources :properties do
 because it is a separate entity and table from properties

Reply:
Thank you for correct me!